staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -40,6 +40,14 @@ import (
 	utiltrace "k8s.io/apiserver/pkg/util/trace"
 )
 
+const (
+	maxBodyLength = (1 << 20) / 8
```

Comment:
https://github.com/kubernetes/kubernetes/pull/72053 changed the [decoder limit](https://github.com/kubernetes/kubernetes/issues/57073#issuecomment-350932992) to 16MB. https://github.com/kubernetes/kubernetes/pull/73805 added a 100MB request body size limit in apiserver

the current etcd limit on object size is [1.5MB](https://github.com/kubernetes/kubernetes/pull/72053#issuecomment-447407483). Both limit changes above are larger than 1.5MB because apiserver can store protobuf in etcd but take/serve JSON (which is larger, but the compression ratio is hard to determine)